proto/ibc/core/client/v1/client.proto

```diff
@@ -98,6 +98,8 @@ message Height {
 
 // Params defines the set of IBC light client parameters.
 message Params {
-  // allowed_clients defines the list of allowed client state types.
+  // allowed_clients defines the list of allowed client state types which can be created
+  // and interacted with. If a client type is removed from the allowed clients list, usage
+  // of this client will be temporarily disabled.
```

Comment:
I think `temporarily` is a bit vague and it doesn't explicitly say what you need to do to enable it again (even though it should be pretty obvious...).

```suggestion
  // of this client will be disabled until it is added again to the list.
```